civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -7431,3 +7431,25 @@ function civihr_employee_portal_html_head_alter(&$head_elements) {
 
   $head_elements[$default_favicon_element]['#attributes']['href'] = "{$base_url}/{$icoPath}";
 }
+
+function get_civihr_version() {
```

Comment:
Please document the function - put short description that the function gets CiviHR version using CiviCRM api and stores it in cache.